src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,5 @@
+@mixin set-colums($min-width, $colums) {
```

Comment:
There is a typo in the parameter name '$colums'. It should be '$columns' to correctly reflect the purpose of setting columns.